deploy/helm/kubecf/templates/ingress.yaml

```diff
@@ -98,7 +98,11 @@ spec:
             backend:
               serviceName: "{{ .Release.Name }}-router"
               servicePort: 443
+
+# If the ingress feature (ingress-controller) is not enabled, create public services to allow
+# public ingress to the cluster.
```

Comment:
Shouldn't this be _below_ the `{{else}}`? Otherwise it's going to get rendered into the YAML files in the wrong condition. Yes, it's a comment, but it's helpful when debugging (I often just render the ops files locally to see what is going on).

Reply:
Oh, right. It indeed should be below.